server/src/main/java/org/elasticsearch/cluster/metadata/IndexNameExpressionResolver.java

```diff
@@ -194,29 +213,43 @@ public IndexNameExpressionResolver(Settings settings) {
             }
 
             Collection<IndexMetaData> resolvedIndices = aliasOrIndex.getIndices();
```

Comment:
can we make if clause have this structure and inline this variable in the second else (see marker):

```
 if (aliasOrIndex.isAlias() && context.isResolveToWriteIndex()) {
    .. write operation path
} else {
  ... read path ..
  start the old code here and put the resolvedIndices variable definition here.
  
}
```

Reply:
ping about inlining this variable.